test/ctr.bats

```diff
@@ -109,6 +109,35 @@ function teardown() {
 	stop_crio
 }
 
+@test "devices" {
+	DEVICES="--additional-devices /dev/null:/dev/xvdc:rwm" start_crio
```

Comment:
how's this test passing if `DEVICES` isn't picked up by helpers.bash? :thinking:

Reply:
you should do something like https://github.com/kubernetes-sigs/cri-o/blob/master/test/helpers.bash#L31 and https://github.com/kubernetes-sigs/cri-o/blob/master/test/helpers.bash#L246 but I'm still wondering how the CI passes here

Reply:
:flushed: just found that the CI passes because I only check `[ "$status" -eq 0 ]`, it is always true no matter what device I used to check here `run crictl exec --sync "$ctr_id" sh -c "[ -e /dev/xvdc ]"`, so I need to check the `$output`

Reply:
I add it to helper.bash. Now I go back to the CI failures. :thinking:   @rhatdan suggested me to use additional devices, so I just leave it as is for now.

Reply:
@rhatdan we do have `default_ulimits`, `default_capabilities`, `default_mounts`, `default_sysctls`, in order to keep naming consistent I propose `default_devices` as they are default devices being added _by default_ for every container